.github/scripts/determine_simd_tag.go

```diff
@@ -3,32 +3,24 @@ package main
 import (
```

Comment:
I had misunderstood the behaviour of the ref/pr values in GithubActions. I simplified this script to simply accept a PR number as a string, or use "main" if we are not in a PR.